modules/consul-cluster/outputs.tf

```diff
@@ -11,11 +11,11 @@ output "cluster_tag_name" {
 }
 
 output "instance_group_url" {
-  value = "${google_compute_instance_group_manager.consul_server.self_link}"
+  value = "${google_compute_region_instance_group_manager.consul_server.self_link}"
 }
 
 output "instance_group_name" {
-  value = "${google_compute_instance_group_manager.consul_server.name}"
+  value = "${google_compute_region_instance_group_manager.consul_server.name}"
```

Comment:
Sanity check. Are `self_link` and `name` valid outputs? They don't appear documented but they may still be valid.